mathjax3-ts/ui/menu/Menu.ts

```diff
@@ -66,16 +66,25 @@ export interface MenuSettings {
     ctrl: boolean;
     shift: boolean;
     scale: string;
+    autocollapse: boolean;
+    collapsible: boolean;
+    inTabOrder: boolean;
+    // A11y settings
+    backgroundColor: string;
+    braille: boolean;
     explorer: boolean;
+    foregroundColor: string;
     highlight: string;
-    background: string;
-    foreground: string;
+    infoPrefix: boolean;
+    infoRole: boolean;
+    infoType: boolean;
+    magnification: string;
+    magnify: string;
     speech: boolean;
-    subtitles: boolean;
     speechrules: string;
-    autocollapse: boolean;
-    collapsible: boolean;
-    inTabOrder: boolean;
+    subtitles: boolean;
+    treecoloring: boolean;
+    viewbraille: boolean;
```

Comment:
Should these be `treeColoring` and `viewBraile` for consistency?  (And `speechRules`?)